spec/unit/filters/filter_form_builder_spec.rb

```diff
@@ -202,17 +202,17 @@ def filter(name, options = {})
       let(:body) { filter :published_date, input_html: { 'autocomplete': "off" } }
 
       it "should generate provided input html for both ends of date range" do
-        expect(body).to have_selector("input.datepicker[name='q[published_date_gteq]'][autocomplete=off]")
-        expect(body).to have_selector("input.datepicker[name='q[published_date_lteq]'][autocomplete=off]")
+        expect(body).to have_css("input.datepicker[name='q[published_date_gteq]'][autocomplete=off]")
+        expect(body).to have_css("input.datepicker[name='q[published_date_lteq]'][autocomplete=off]")
```

Comment:
There are a few of these in this file (at least) but I saw that it's using `have_css` vs `have_field`. Shouldn't this be using the latter with the class as a keyword argument?

Reply:
I was not able to find a way to test for `autocomplete` with `have_field` matcher